eng/InstallRuntimes.proj

```diff
@@ -146,6 +146,21 @@
     <Copy SourceFiles="@(PrivateBuildFiles)" DestinationFolder="$(DotNetInstallDir.Replace('\\', '\'))" />
   </Target>
 
+<!--
+    Removes the private build registry keys
+-->
+
+  <Target Name="CleanupPrivateBuild"
+          Condition="Exists($(RemoveRegeditFileName))">
+
+    <Exec Command="regedit $(RemoveRegeditFileName)" />
+    <!-- 
+        Deleting only the add reg key file so ModifyRegistry can run again. Leaving the
```

Comment:
```suggestion
        Deleting only the add reg key file so ModifyRegistry can't run again. Leaving the
```

Reply:
It probably isn't clear, but I do mean "can".  Deleting the AddPrivateBuild.reg file allows the ModifyRegistry target to run again creating the reg file and regedit'ing it.